css/index.css

```diff
@@ -1,3 +1,452 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
 
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
 
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 20px;
+}
+
+.container {
+  width: 885px;
+  margin: 0 auto;
+  border: 3px grey;
+}
+
+header {
+  width: 100%;
+  display: flex;
+}
+
+.image-wrapper {
+  display: none;
+}
+
+header nav {
+  width: 800px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+header nav a {
+  margin-right: 0 auto;
+  font-size: 18px;
+  color: grey;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 10px;
+}
+.gateway {
+  display: flex;
+  flex-direction: row;
+}
+
+.gateway-text {
+  display: flex;
+  flex-direction: column;
+  vertical-align: center;
+  font-size: 55px;
+  margin-top: 100px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.gateway .gateway-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 8px;
+  padding: 8px 50px;
+  font-size: 18px;
+  border: 3px solid grey;
+  cursor: pointer;
+}
+
+.gateway .gateway-text button:hover {
+  background: grey;
+  color: yellow;
+}
+
+/* media query mobile phone begin */
+@media (max-width: 500px) {
+  .container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  header {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  header nav a {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-top: 40px;
```

Comment:
you can combine margin top and margin bottom on one line like: margin: 40px 0 5px 0;